contxt/models/ems.py

```diff
@@ -42,6 +42,62 @@ class MainService(ApiObject):
     updated_at: datetime
 
 
+@dataclass
+class Metric(ApiObject):
```

Comment:
do you know why this api has its own concept of a `Metric` and `MetricValue`? i thought they would be specific to the assets api

Reply:
EMS is now a proxy for the Assets API as the latter is phased out.

Reply:
gotcha